src/icons/SealEye.tsx

```diff
@@ -1,9 +1,63 @@
-import { stroke } from 'classnames/tailwind'
+import classnames, { stroke, strokeWidth } from 'classnames/tailwind'
 
-const strokeSecondaryDimmed = stroke('stroke-secondary-dimmed')
-const strokeTertiaryLine = stroke('stroke-tertiary')
+enum SealEyeColor {
+  accent = 'stroke-accent',
+  secondary = 'stroke-secondary-dimmed',
+  tertiary = 'stroke-tertiary',
+}
+
+const commonStrokeWidth = strokeWidth('stroke-1.5')
+const strokeClass = (color: SealEyeColor) =>
```

Comment:
Why don't you put the function inside the component, so you don't need to call it each time, only use the result?

Reply:
Did it